ll

```diff
@@ -0,0 +1,1215 @@
+/home/ydayagi/work/go/src/github.com/ydayagi/k4e-operator/bin/controller-gen "crd:trivialVersions=true,preserveUnknownFields=false" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
```

Comment:
I think that file has gotten into the PR by accident.

Reply:
thanks